ext/dba/dba_lmdb.c

```diff
@@ -43,10 +43,18 @@ DBA_OPEN_FUNC(lmdb)
 	MDB_env *env;
 	MDB_txn *txn;
 	int rc, mode = 0644, flags = MDB_NOSUBDIR;
+	zend_long mapsize = 0;
 
 	if(info->argc > 0) {
 		mode = zval_get_long(&info->argv[0]);
 
+		if (info->argc > 1) {
+			mapsize = zval_get_long(&info->argv[1]);
+			if (mapsize < 0) {
```

Comment:
`mapsize` may be equal 0

Reply:
`mapsize==0` is supposed to be the default (which also may be passed explicitly), in which case no warning should be emitted. Do you suggest to change the default to  `-1`, and let the user pass `0` to have a mapsize of zero?

Reply:
No, I thought it was a mistake :)